scripts/render_DESeq2_report.R

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/R
```

Comment:
So I think this file can be deleted in your branch right? Is there anything in here we didn't capture in the parallel one? I think we're good but just double checking

Reply:
Oops, yes thanks for the reminder.